index.js

```diff
@@ -32,7 +32,8 @@ function underPressure (fastify, opts, next) {
           type: 'object',
           properties: {
             status: { type: 'string' }
-          }}
+          },
+          additionalProperties: false}
```

Comment:
This is not needed, we are not sending anything more.
Furthermore by using `additionalProperties` you are forcing `fast-json-stringify` to check if there are additional properties.

Reply:
Truly `fast-json-stringify` doens't make that check (see [here](https://github.com/fastify/fast-json-stringify/blob/master/index.js#L371))

And by [default](https://spacetelescope.github.io/understanding-json-schema/reference/object.html#properties) `additionalProperties` is true

Anyway `fast-json-stringify` explicits that [missing fields aren't serialized](https://github.com/fastify/fast-json-stringify#missing-fields). But it is just an implementation detail.

Reply:
Yes, but again this is really not needed.
The output that we are sending back is always the same, it's hard coded! :P
Let's not make things harder than they are ;)